docs/blog/newsletter-november-2017.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: November 05, 2017
+   :tags: newsletter
+
+
+#########################################
+Write the Docs Newsletter – November 2017
+#########################################
+
+Happy November everybody! Kelly O. here, reporting live from the coffee shop that is about to cut me off because I've had three flat whites in a row. But I'm considering them celebratory beverages, because we have a jam-packed month of documentation goodness to talk about.
+
+In the next few weeks, the orangization team will be:
+
+- Announcing the call for proposals (and looking for volunteers) for next year's Portland conference! (If you're not signed up to the conference mailing list, you can do so in `the sidebar of the WTD homepage <http://www.writethedocs.org/>`_.)
+- Wishing we were all going to be in Australia for their `Write the Docs Day mini-conference <http://www.writethedocs.org/conf/au/2017/>`_.
+- Looking on excitedly as API the Docs holds `their second one-day event in Amsterdam <http://apithedocs.org/>`_.
+- Musing on how we got so lucky as to have almost a dozen meetups this month in a cities across the western hemisphere. Scroll down for details!
```

Comment:
Internal link? Or can't we?

Reply:
I believe I can add it manually in MailChimp, but I haven't been able to figure out how to do it in rst.

Reply:
If you link to `#upcoming-community-events` it should work? 

Or even `Upcoming community events`_